test/inductor/test_torchinductor.py

```diff
@@ -1485,6 +1485,13 @@ def fn(a, b, c):
             ),
         )
 
+    # https://github.com/pytorch/pytorch/issues/98979
+    @unittest.skipIf(HAS_CUDA, "cuda failed for float64 linear")
```

Comment:
I skip this case for cuda path which it is always failed.  @jansel @desertfire @ngimel